prep-javascript-objects/main.js

```diff
@@ -0,0 +1,15 @@
+var person = {
+  firstName: 'Mayerly ',
+  lastName: 'Henao',
+  hobby: 'Under Water Rugby',
+  job: 'Data Analyst',
+  previousJob: 'Manager at a Brewery'
+};
```

Comment:
rather than modifying the object here, we can update it and add properties later in our code.

```js
person.job = 'Data Analyst';
```

Reply:
<img width="564" alt="Screenshot 2022-11-29 at 3 53 24 PM" src="https://user-images.githubusercontent.com/64482290/204645983-074f57c4-e196-4379-a6b6-4d3083291f75.png">

Reply:
<img width="709" alt="Screenshot 2022-11-29 at 4 00 30 PM" src="https://user-images.githubusercontent.com/64482290/204646722-2e3174bb-af5e-4b87-94d0-5a0ceda21330.png">